gems/pending/Gemfile

```diff
@@ -23,7 +23,9 @@ gem "httpclient",              "~>2.7.1",           :require => false
 gem "image-inspector-client",  "~>1.0.3",           :require => false
 gem "iniparse",                                     :require => false
 gem "kubeclient",              "=1.2.0",            :require => false
-gem "linux_admin",             "~>0.17.0",          :require => false
+gem "linux_admin",             "~>0.18.0",          :require => false
+gem "sshkey",                  "~>1.8.0",           :require => false
```

Comment:
@jrafanie Tests were failing because it couldn't 'require sshkey' in (https://github.com/ManageIQ/manageiq/pull/7517/files#diff-acdc70b50bcc2b0000210de787454f68R209), i added it over here, are the pending gems tests isolated from the main Gemfile? should i keep both or just move it to pending?